task/04-arrays-tasks.js

```diff
@@ -536,9 +638,12 @@ function selectMany(arr, childrenSelector) {
  *   [[[ 1, 2, 3]]], [ 0, 0, 1 ]      => 2        (arr[0][0][1])
  */
 function getElementByIndexes(arr, indexes) {
-    throw new Error('Not implemented');
-}
+   var arrayStringIndex = indexes.reduce(function(total, currentValue) {
+      return total + '['+ currentValue + ']';
+    }, '');
 
+  return eval("arr" + arrayStringIndex);
+}
 
```

Comment:
`eval` is not a fine solution, see https://javascript.info/eval
you can do that with .reduce only